src/main/java/com/javahelps/restservice/CodeReview.java

```diff
@@ -1 +1,42 @@
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
 
+public class CodeReview {
+
+    Integer x;
```

Comment:
indicate type of encapsulation of the variable accessing modifiers like ( public,private )